packages/components/psammead-inline-link/README.md

```diff
@@ -0,0 +1,92 @@
+# psammead-inline-link &middot; [![GitHub license](https://img.shields.io/badge/license-Apache%202.0-blue.svg)](https://github.com/BBC-News/psammead/blob/latest/LICENSE) [![npm version](https://img.shields.io/npm/v/@bbc/psammead-inline-link.svg)](https://www.npmjs.com/package/@bbc/psammead-inline-link) [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg)](https://reactjs.org/docs/how-to-contribute.html#your-first-pull-request)
+
+## Description
+
+The `@bbc/psammead-inline-link` package exports a single Inline Link component. It uses an `a` HTML element and colours from `@bbc/psammead-styles`.
+
+## Installation
+
+```
+npm install @bbc/psammead-inline-link --save
+```
+
+## Usage
+
+```jsx
+import InlineLink from '@bbc/psammead-inline-link';
+
+const WrappingComponent = () => (
+  <InlineLink href="https://www.bbc.com/news">Text here</InlineLink>
+);
+```
+
+## When to use this component
+
+This component can be used at any point on a page.
+
+We have not included typography styles in with this component. This is so it is resusable in multiple contexts. You can use our existing `psammead-paragraph` for body copy and `psammead-headings` for headlines and subheadings (`h1`s and `h2`s), in this way:
+
+In a paragraph:
+
+```jsx
+import InlineLink from '@bbc/psammead-inline-link';
+import Paragraph from '@bbc/psammead-paragraph';
+
+const WrapperComponent = () => (
+  <Paragraph>I am a paragraph that has a link to <InlineLink href="https://www.bbc.com/news">the BBC News front page</InlineLink>.<Paragraph>
+);
+```
+
+In a Headline or SubHeading:
+
+```jsx
+import InlineLink from '@bbc/psammead-inline-link';
+import { Headline, SubHeading } from '@bbc/psammead-headings';
+
+const HeadlineWithALink = () => (
+  <Headline><InlineLink href="https://www.bbc.com/news">Headline</InlineLink>
+  <Headline>
+
+  const SubHeadingWithALink = () => (
+  <SubHeading><InlineLink href="https://www.bbc.com/news">SubHeading</InlineLink><SubHeading>
```

Comment:
Based on our examples on BBC News, I'd invert these (i.e. `<InlineLink><SubHeading>foo</SubHeading></InlineLink>`). I think we nest the headings inside the links for accessibility reasons (perhaps so that screen readers can navigate by headings and easily click the link if interested?)